.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    singleQuote: true,
+    trailingComma: 'es5',
+    printWidth: 120,
+    jsxBracketSameLine: true
```

Comment:
```suggestion
    jsxBracketSameLine: true,
    endOfLine: 'auto'
```
https://stackoverflow.com/a/53769213